h_periodic/views/status.py

```diff
@@ -1,20 +1,20 @@
 import os
+from os import environ
 
 import kombu
 import psutil
 from kombu.exceptions import KombuError
-from pyramid.view import view_config, view_defaults
+from pyramid.view import view_config
 
 
-@view_defaults(route_name="status", renderer="json", request_method="GET")
 class StatusView:
     H_PID_FILE = "celerybeat.pid"
     H_BROKER_VAR = "BROKER_URL"
 
     def __init__(self, _context, request):
         self.request = request
 
-    @view_config
+    @view_config(route_name="status", renderer="json", request_method="GET")
```

Comment:
Fixing a suggestion in the PR. Turns out you do need the braces, but there's no point in having defaults for only one view